site/static/main.css

```diff
@@ -260,6 +261,10 @@ nav a:hover {
   text-decoration: none;
 }
 
+.title h1 {
+  margin-top: 0.3em;
```

Comment:
Not sure why this is needed. Why is this different than any other page on our website?